src/Footer/Footer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Row } from 'antd';
+
+const StyledRow = styled(Row)`
+  text-align: center;
+  position: absolute;
+  bottom: 0;
+  width: 100vw;
+`;
+const Footer = styled.div`
+  text-align: center;
+`;
+
+const FooterComponent = () => {
+  return (
+    <StyledRow style={{ textAlign: 'center' }}>
```

Comment:
You are using styled-components then what is the need of inline-styles?

Reply:
Updated